### ISSUE TITLE
Bad session management

### ISSUE THREAD

**Issue by kevr** [kevr/breakscheduler#9]
In Firefox, this is a lot more apparent, but the problem also exists on Google Chrome and presumably most other browsers in existence.

When refreshing the website at the login page, we get conflicting behavior. Sometimes it logs in, sometimes it does not. It doesn't always remember data.

I believe this issue may have something to do with our Barrier components contacting the API with localStorage `@authToken` before deciding whether or not it should login.

This needs to be solved before we can merge into `master`.

**Comment by kevr** [comment]
I cannot simply reproduce this randomly.

Part of the issue with possible reproduction is: `@rememberedLogin` is somewhat tied to `@authToken`, in that, our authentication router wall attempts to use `@authToken` to login before `@rememberedLogin` is ever attempted via `help/support/Login`'s componentDidMount case.

**Comment by kevr** [comment]
Deferred the importance of this issue due to not being completely reproducible. This is a nice to-have, and should be done before the website is in production, however it is more pressing to complete our page content.
